examples/blog/mailSubscription/src/pages/mail/create.tsx

```diff
@@ -40,6 +40,7 @@ export const MailCreate: React.FC<IResourceComponentsProps> = () => {
                     <ReactMde
                         selectedTab={selectedTab}
                         onTabChange={setSelectedTab}
+                        // eslint-disable-next-line
                         generateMarkdownPreview={(markdown: any) =>
```

Comment:
```suggestion
                        generateMarkdownPreview={(markdown) =>
```

We can just remove `any` since `react-mde` already comes with type definitions.

Reply:
done